Resources/Textures/Mobs/Customization/human_hair.rsi/meta.json

```diff
@@ -567,6 +567,10 @@
         "name": "protagonist",
         "directions": 4
       },
+      {
+        "name": "pulato",
```

Comment:
Please add an attribution to the `copyright` line at the top of the file.
For example "pulato.png made by DreamlyJack for SS14". If you did not make it yourself or made it by modifying an existing sprite, then make sure to mention that.

Reply:
ok
thanks